module_1/module_1/Program.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+
+namespace Module_1
+{
+    internal class Program
+    {
+        private static void Main()
+        {
+            int n = Convert.ToInt32(Console.ReadLine());
+            int[] main_Array = new int[n];
+            int[] first_Number_Array = new int[n];
+            int[] second_Number_Array = new int[n];
+            char[] first_Letter_Array = new char[n];
+            char[] second_Letter_Array = new char[n];
+            int first_amount = 0;
+            int second_amount = 0;
+
+            for (int i = 0; i < n; i++)
```

Comment:
The code can be separated into the standalone methods (functions), so it would be much easier to read and understand